CMakeLists.txt

```diff
@@ -58,17 +62,22 @@ set(CMAKE_CXX_EXTENSIONS ON)
 set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
 
 include(
-  "${CMAKE_CURRENT_LIST_DIR}/tools/cmake/helpers/git_commit_information.cmake")
+    "${CMAKE_CURRENT_LIST_DIR}/tools/cmake/helpers/git_commit_information.cmake"
+)
 
 # Jenkins integration section dont modify this section if you dont know details
 # about integration with Jenkins!!!
 set(HMI "web" CACHE STRING "HMI type")
 option(
-  EXTENDED_MEDIA_MODE
-  "Turn on and off extended Madia Manager features relates to PulseAudio A2DP and GStreamer"
-  OFF)
-option(BUILD_SHARED_LIBS
-       "Build all libraries as shared (if ON) or static (if OFF)" OFF)
+    EXTENDED_MEDIA_MODE
```

Comment:
@MaximRybak  please write `option` and  `EXTENDED_MEDIA_MODE` in one line

Reply:
@BSolonenko We must ignoring formatting "option" command ?

Reply:
@MaximRybak It is bad formatting. please fix it